src/status_im/integration_test.cljs

```diff
@@ -188,42 +164,13 @@
              [:my-profile/finish-success]
              (is (nil? @(rf/subscribe [:mnemonic]))) ; assert seed phrase has been removed
              (logout!)
-             (rf-test/wait-for [::logout/logout-method]
-               (assert-logout)))))))))
+             (rf-test/wait-for [::logout/logout-method]))))))))
 
 (def multiaccount-name "Narrow Frail Lemming")
 (def multiaccount-mnemonic
   "tattoo ramp health green tongue universe style vapor become tape lava reason")
 (def multiaccount-key-uid "0x694b8229524820a3a00a6e211141561d61b251ad99d6b65daf82a73c9a57697b")
 
-(deftest recover-multiaccount-test
```

Comment:
why is this test being nuked can you elaborate?

Reply:
in next part i'll nuke all old code so there will be more